staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/accessors_test.go

```diff
@@ -37,7 +38,7 @@ func TestMutatingWebhookAccessor(t *testing.T) {
 			orig.ReinvocationPolicy = nil
 
 			uid := fmt.Sprintf("test.configuration.admission/%s/0", orig.Name)
-			accessor := NewMutatingWebhookAccessor(uid, "test.configuration.admission", orig)
+			accessor := NewMutatingWebhookAccessor(uid, "test.configuration.admission", logicalcluster.New(""), orig)
```

Comment:
(General comment, not line specific)

Would like to ensure the unit test accounts for the new GetLogicalCluster function. I see 2 options:

1. We modify this file
2. We create a new `kcp_accessors_test.go` file where the only thing we test is our changes

1 keeps everything together. 2 makes rebases easier. I think I maybe am voting for 2.

@sttts WDYT?

Reply:
external file if possible